commit c42fd7b5ad5dce830e7f7238a76568d8e3e2d917 (refs/changes/73/2173/2)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2017-06-30T19:05:47+00:00 (2 years, 3 months ago)
    
    FWAPI-287 firewaller appears to be leaving thousands of zombie LWPs behind
    Reviewed by: Nick Zivkovic <nick.zivkovic@joyent.com>
    Approved by: Nick Zivkovic <nick.zivkovic@joyent.com>

diff --git a/package.json b/package.json
index 7ab07a4..61ef70a 100644
--- a/package.json
+++ b/package.json
@@ -11,7 +11,7 @@
         "clone": "1.0.2",
         "fast-stream": "git+https://github.com/joyent/sdc-fast-stream.git#893d59d",
         "jsprim": "0.5.0",
-        "lockfd": "1.0.1",
+        "lockfd": "~1.2.1",
         "restify": "2.6.1",
         "sdc-clients": "10.2.0",
         "uuid": "3.0.1",
